internal/controller/datadogagent/controller_reconcile_v2_helpers.go

```diff
@@ -94,15 +94,17 @@ func (r *Reconciler) reconcileAgentProfiles(ctx context.Context, logger logr.Log
 	profiles := []datadoghqv1alpha1.DatadogAgentProfile{{}}
 	metrics.IntrospectionEnabled.Set(metrics.FalseValue)
 	metrics.DAPEnabled.Set(metrics.FalseValue)
-
+	logger.Info("reconcileAgentProfiles", "DatadogAgentProfileEnabled", r.options.DatadogAgentProfileEnabled, "IntrospectionEnabled", r.options.IntrospectionEnabled)
 	// If profiles or introspection is enabled, get the node list and update providers.
 	if r.options.DatadogAgentProfileEnabled || r.options.IntrospectionEnabled {
 		nodeList, err := r.getNodeList(ctx)
 		if err != nil {
 			return reconcile.Result{}, err
 		}
+		logger.Info("nodeList", "nodeList", nodeList)
```

Comment:
we will need to remove logs or change level before merging.

Reply:
fixed in EKS PR #2031 https://github.com/DataDog/datadog-operator/pull/2031/commits/a75a591564761f4cc6499a0452757e79709629ef and https://github.com/DataDog/datadog-operator/pull/2031/commits/75319bb6d937b04fe827eeb350f571f4269d2c79